From 5b3ee107c549a4542a0048000e5b345a2c00bf33 Mon Sep 17 00:00:00 2001
From: Robert Nelson <robertcnelson@gmail.com>
Date: Wed, 2 Nov 2011 12:56:24 -0500
Subject: [PATCH] ARM: OMAP: igep00x0 fix build of igep00x0_wifi_bt_init when LIBERTAS_SDIO is disabled in config

Noticed by Misha Manulis: https://github.com/RobertCNelson/stable-kernel/issues/4

When I'm compiling, I get the following error:

  arch/arm/mach-omap2/board-igep00x0.c:488:13: error: conflicting types for ‘igep00x0_wifi_bt_init’

The fix is to patch __KERNEL/arch/arm/mach-omap2/board-igep00x0.c:488__ to replace that line with the following:

   void igep00x0_wifi_bt_init(int npd, int wifi_nreset, int bt_nreset, int bt_enable) { }

Signed-off-by: Robert Nelson <robertcnelson@gmail.com>
---
 arch/arm/mach-omap2/board-igep00x0.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/mach-omap2/board-igep00x0.c b/arch/arm/mach-omap2/board-igep00x0.c
index 901b8f7..8e5dc09 100644
--- a/arch/arm/mach-omap2/board-igep00x0.c
+++ b/arch/arm/mach-omap2/board-igep00x0.c
@@ -485,7 +485,7 @@ void __init igep00x0_wifi_bt_init(int npd, int wifi_nreset, int bt_nreset,
 }
 
 #else
-void __init igep00x0_wifi_bt_init(int npd, int wifi_nreset, int bt_nreset) { }
+void __init igep00x0_wifi_bt_init(int npd, int wifi_nreset, int bt_nreset, int bt_enable) { }
 #endif
 
 #if defined(CONFIG_EEPROM_AT24) || \
-- 
1.7.4.1

